app/models/follow_request.rb

```diff
@@ -0,0 +1,25 @@
+# == Schema Information
+#
+# Table name: follow_requests
+#
+#  id           :bigint           not null, primary key
+#  status       :string           default("pending")
+#  created_at   :datetime         not null
+#  updated_at   :datetime         not null
+#  recipient_id :bigint           not null
+#  sender_id    :bigint           not null
+#
+# Indexes
+#
+#  index_follow_requests_on_recipient_id  (recipient_id)
+#  index_follow_requests_on_sender_id     (sender_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (recipient_id => users.id)
+#  fk_rails_...  (sender_id => users.id)
+#
+class FollowRequest < ApplicationRecord
+  belongs_to :recipient, class_name: "User"
+  belongs_to :sender, class_name: "User"
```

Comment:
You could greatly benefit from having an `enum` for the `status`. Can you add this?